celesta-sql/src/test/java/ru/curs/celesta/score/ParserTest.java

```diff
@@ -63,6 +63,27 @@ private void parseAndSaveCsqlScript(String csqlScript, File grainPath, String gr
       g.save();
     }
   }
+  
+  @Test
```

Comment:
This test is insufficient. You check that you parse the `WITH NO AUTOUPDATE` without any errors, but you never `assertFalse(grain.isAutoupdate)`. 

It is also desirable to check for correct parsing the `WITH AUTOUPDATE` clause (`assertTrue(grain.isAutoupdate)`) and the default behaviour (no `WITH` clause, assertTrue(grain.isAutoupdate)).

See examples of such tests in `test1`...`test4` (yeah I know, this is awful, but they are very old tests ;-) )

Together with `SchemaAutoupdateTest` this is going to be OK.

Reply:
A separate test `SchemaAutoupdateTest.testScoreParsing()` added.